core/frontend/src/components/video-manager/VideoStreamCreationDialog.vue

```diff
@@ -274,6 +274,12 @@ export default Vue.extend({
       }
       return true
     },
+    no_repetitions(input: string): (true | string) {
+      if (this.stream_endpoints.filter((x) => x === input).length > 1) {
```

Comment:
`!this.stream_endpoints.filter((x) => x === input).isEmpty()`

Reply:
it's checking for more than one, not more than zero